CHANGELOG.md

```
@@ -43,6 +43,10 @@ Other guiding principles:
 - **workflows**: executable permissions are now correctly preserved in the release workflow.
 - **amaru-ledger**: Correctly calculate an output's minimum lovelace value.
 
+### Fixed
```

markdownlint-cli2 (0.23.2)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)